recipes/octo-encryption-cpp/all/conanfile.py

```diff
@@ -0,0 +1,97 @@
+from conans import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
```

Comment:
```suggestion
from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
```
Need to generate openssl cmake files

Reply:
done